components/DocsLayout.js

```diff
@@ -1,27 +1,65 @@
-import React from 'react'
+import React, { Component } from 'react'
 import Head from './SeoHead'
-import Navbar from './Navbar'
+import Nav from './Nav'
 import { Container, Content, Title } from './Layout'
 
-const DocsLayout = ({ children, title="", description="" }) => (
-  <Container>
 
-    <Head
-      title={`styled-components${title ? `: ${title}` : ''}`}
-      description={description}>
-        <meta name="robots" content="noodp" />
-    </Head>
+class DocsLayout extends Component {
+  state = {
+    isSideFolded: true
+  }
 
-    <Navbar />
+  static defaultProps = {
+    title: '',
+    description: '',
+  }
 
-    <Content>
-      <Title>
-        {title}
-      </Title>
+  onSideFold = () => {
+    let isSideFolded = !this.state.isSideFolded
+    // if(!isSideFolded) {
+    //   document.body.classList.add('sticky')
+    // } else {
+    //   document.body.classList.remove('sticky')
+    // }
+    this.setState({
+      isSideFolded: isSideFolded
```

Comment:
Small nit, can you remove the variable above? Also it won't need to be a `let`. We generally use the `const` before `let` convention

Reply:
Yeah, you're right, I forgot to refactor the old code here. Sorry!